src/GraphEditor.jsx

```diff
@@ -60,7 +65,9 @@ export default class GraphEditor extends React.Component {
                         className="btn btn-danger float-md-right"
                         title="Delete Graph"
                         href={`/course/${courseId}/graph/${this.props.gId}/delete/`}>
-                         <svg height="32" className="octicon octicon-x" viewBox="0 0 12 16" version="1.1" width="24" aria-hidden="true"><path fillRule="evenodd" d="M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48L7.48 8z"></path></svg>
+                        <svg alt="" height="32" className="octicon octicon-x" viewBox="0 0 12 16" version="1.1" width="24" aria-hidden="true">
```

Comment:
Again - is an empty alt tag helpful?

Reply:
Yeah, the X octicon is purely decorative, same for the Clone icon above.